libs/community/tests/integration_tests/indexes/docker-compose/mongodb.yml

```diff
@@ -0,0 +1,11 @@
+version: '3.1'
```

Comment:
Would you be able to move this to langchain/docker/docker-compose.yml?

Reply:
Sure, I'll do that. Thanks for the review!